ivy.el

```diff
@@ -3954,10 +3954,10 @@ When `ivy-calling' isn't nil, call `ivy-occur-press'."
     (ivy-occur-press)))
 
 (define-derived-mode ivy-occur-mode fundamental-mode "Ivy-Occur"
-  "Major mode for output from \\[ivy-occur].
+                     "Major mode for output from \\[ivy-occur].
 
 \\{ivy-occur-mode-map}"
-  (setq-local view-read-only nil))
+                     (setq-local view-read-only nil))
```

Comment:
`define-derived-mode` isn't even Common Lisp; the forms being indented are a docstring and body, not distinguished arguments. See [`(elisp) Indenting Macros`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Indenting-Macros.html).